bin/build

```diff
@@ -24,6 +27,7 @@ source /opt/nvm/nvm.sh
 # --json || --debug
 
 cd $INPUT_DIR
+printf "Using $(meteor --version)"
```

Comment:
`Using Meteor 1.3Demeteorizing application...`

Need that `/n` if you want to use `printf`...